.gitignore

```diff
@@ -11,7 +11,7 @@ __pycache__*
 Untitled*.ipynb
 
 # data related dirs
-data/
+# data/
```

Comment:
I have a local dir I used to call `data` (that cannot be called like this anymore🙂) 

Could you include here `tmp_dir` (and perhaps other names you might want to use if any) so that it looks like this

```
# data related dirs
tmp_data/
model_weights/
tmp_dir/
weights/

# Unit Tests/Coverage
...
```

If not is ok, I can just locally add it to my `.gitignore` file

Reply:
Added it myself